Makefile

```diff
@@ -0,0 +1,26 @@
+export SHELL:=/bin/bash
+KONG_DIST_KUBERNETES_VERSION?=master
+KONG_BUILD_TOOLS_VERSION?=master
+
+setup_tests:
+	-rm -rf kong-build-tools
+	git clone https://github.com/Kong/kong-build-tools.git
+	cd kong-build-tools; \
+	git fetch; \
+	git reset --hard origin/$(KONG_BUILD_TOOLS_VERSION); \
+	make setup_tests
+
+test:
+	-rm -rf kong-dist-kubernetes
+	git clone https://github.com/Kong/kong-dist-kubernetes.git
+	cd kong-dist-kubernetes; \
+	git fetch; \
+	git reset --hard origin/$(KONG_DIST_KUBERNETES_VERSION)
+	kubectl apply -f https://raw.githubusercontent.com/Kong/kong-build-tools/$(KONG_BUILD_TOOLS_VERSION)/kube-registry.yaml
```

Comment:
Don't we already have kong-build-tools cloned? Lets use the local copy instead of relying on github?

Also would need double quotes...